src/contracting/storage/orm.py

```diff
@@ -38,6 +38,10 @@ def __init__(self, contract, name, driver: Driver = driver, default_value=None):
         self._delimiter = constants.DELIMITER
         self._default_value = default_value
 
+        # Store the default_value in storage if it's not None
+        if default_value is not None:
+            self._driver.set(f'{self._key}{self._delimiter}__default__', default_value)
```

Comment:
should `__default__` be in a constants file or similar somewhere ?

Reply:
Yea probably a good idea. I will take another look at it